### PR TITLE
Upgrade to React 0.13.3

### PR DESCRIPTION
**Pull request by tjphopkins**

### CODE REVIEW

File: dist/react-simpletabs.js [hunk 1]
```diff
@@ -75,71 +75,68 @@ return /******/ (function(modules) { // webpackBootstrap
...
 	    onBeforeChange: React.PropTypes.func,
 	    onAfterChange: React.PropTypes.func,
-	    children: React.PropTypes.oneOfType([
-	      React.PropTypes.array,
-	      React.PropTypes.element
-	    ]).isRequired
+	    children: React.PropTypes.oneOfType([React.PropTypes.array, React.PropTypes.element]).isRequired
 	  },
-	  getDefaultProps:function () {
+	  getDefaultProps: function getDefaultProps() {
```
Comment (gbrady92) [inline dist/react-simpletabs.js:85]: More a question than a comment:

What is the difference between `getDefaultProps:function () {` and `getDefaultProps: function getDefaultProps() {`

Comment (tjphopkins) [reply dist/react-simpletabs.js:85]: God knows. This is js file built from the jsx by Babel

Comment (tjphopkins) [reply dist/react-simpletabs.js:85]: This is the jsx:

```
 getDefaultProps () {
    return { tabActive: 1 };
  },
```

**Comment by gbrady92** [comment]
🐳  Look's fine to me.

**Pull request merged**